export_dagmc_cmd/iGeom_test.hpp

```diff
@@ -0,0 +1,27 @@
+#ifndef IGEOM_TEST_HPP
+#define IGEOM_TEST_HPP
+
+#include "CubitCommandInterface.hpp"
+#include "CubitMessageHandler.hpp"
+
+// CGM includes
+#include "RefEntity.hpp"
+
+class iGeom_test: public CubitCommand
+{
+public:
+  iGeom_test();
+  ~iGeom_test();
+
+  std::vector<std::string> get_syntax();
+  std::vector<std::string> get_syntax_help();
+  std::vector<std::string> get_help();
+  bool execute(CubitCommandData &data);
+
+private:
+  double radius;
```

Comment:
Hrmm I'm seeing now that the test class can take in the radius and radius2 variables as arguments in which case it makes sense for them to be attributes of the class, but is there a reason for this? 

It seems as though it only opens the tests up to situations in which a user might pass invalid radii for the test scenarios. Maybe this was a learning exercise for arguments to Trelis console commands...